test/e2e/_move.js

```diff
@@ -132,13 +132,7 @@ export async function checkNoUpdateLink(page) {
   return moveDetailPage.checkNoUpdateLink(page)
 }
 
-const updatePages = [
-  'personal_details',
-  'risk',
-  'health',
-  'court',
-  'move_details',
-]
+const updatePages = ['personal_details', 'risk', 'health', 'court', 'date']
```

Comment:
The dramatic changes in formatting when you rename / remove an element are always funny.